src/common/enums.ts

```diff
@@ -19,3 +19,15 @@ export enum SourceType {
   FS = 'FS',
   GPKG = 'GPKG',
 }
+
+// todo - mutual enums with cache-seeder repo - should consider one source
+export enum SeedMode {
+  SEED = 'seed',
+  CLEAN = 'clean',
+}
+
+export enum CacheType {
```

Comment:
MapServerCacheType

Reply:
done

Reply:
changed to MapServerCacheSource. because MapServerCacheType already used in overseer for source definitions (FS or S3)

Reply:
changed:
mapproxy caches type will be called: MapServerCacheType
the internal system source type FS or S3: MapServerCacheSource